runtime/CMakeLists.txt

```diff
@@ -130,19 +119,11 @@ add_dependencies(clean_all clean_runtime)
 ###############################################################################
 set(RUNTIME_FILES_TO_INSTALL)
 
-# This is quite fragile and depends on knowledge in the bitcode
-# build system. Hopefully it won't change very often though.
-
-# FIXME: This hack needs to die!
-if (USE_RUNTIME_BINARY_TYPE_HACK)
-  list(APPEND RUNTIME_FILES_TO_INSTALL
-    "${KLEE_RUNTIME_DIRECTORY}/kleeRuntimeIntrinsic.bc"
-    "${KLEE_RUNTIME_DIRECTORY}/klee-libc.bc")
-else()
-  list(APPEND RUNTIME_FILES_TO_INSTALL
-    "${KLEE_RUNTIME_DIRECTORY}/libkleeRuntimeIntrinsic.bca"
-    "${KLEE_RUNTIME_DIRECTORY}/libklee-libc.bca")
-endif()
+list(APPEND RUNTIME_FILES_TO_INSTALL
+  "${KLEE_RUNTIME_DIRECTORY}/kleeRuntimeIntrinsic.bc"
+  "${KLEE_RUNTIME_DIRECTORY}/klee-libc.bc"
+  "${KLEE_RUNTIME_DIRECTORY}/kleeRuntimeFreeStanding.bc"
```

Comment:
While my last comment on this change was relevant, I now realize that I completely missed the point here: We do no longer generate module but archive files, so the file extension is wrong on all three. But looking at the code you removed, it seems like it was broken before.

Reply:
No, it was not broken - it was depending on the version of LLVM and had different behaviour according to what will be linked.

Reply:
Sorry, I should have expressed more clearly what I was actually trying to hint at: We no longer generate a `kleeRuntimeIntrinsic.bc` but rather a `kleeRuntimeIntrinsic.bca` file. Thus, the file extensions need to be changed from `.bc` to `.bca` here (if I'm not mistaken).

Other than that, you are right, I was wrong about it being broken before this change.